jwst/tests_nightly/general/fgs/test_guider_pipeline.py

```diff
@@ -0,0 +1,89 @@
+import os
+import pytest
+from astropy.io import fits as pf
+from jwst.pipeline.calwebb_guider import GuiderPipeline
+
+
+pytestmark = [
+    pytest.mark.usefixtures('_jail'),
+    pytest.mark.skipif(not pytest.config.getoption('bigdata'),
+                       reason='requires --bigdata')
+]
+
+
+def test_guider_pipeline1(_bigdata):
+    """
+
+    Regression test of calwebb_guider pipeline performed on ID-image data.
```

Comment:
Future work: In the spirit of documenting tests, as brought up by @mdlpstsci , developers should have more details on what this test is supposed to accomplish.